src/components/Base/Image/index.jsx

```diff
@@ -39,21 +41,33 @@ export default class Image extends React.Component {
   }
 
   render() {
-    const { src, className, iconSize, ...rest } = this.props;
+    const { src, iconLetter, className, iconSize, ...rest } = this.props;
     const { failed } = this.state;
 
     if (failed) {
       const nonIcon = '/none.svg';
-      const sizeStyle = {
+      const style = {
         width: iconSize + 'px',
         height: iconSize + 'px'
       };
+      const letter = iconLetter.substr(0, 1).toLocaleUpperCase();
```

Comment:
refactor as 
```
iconLetter[0].toLocaleUpperCase()
```

Reply:
ok

Reply:
if iconLetter is '', iconLetter[0] is undefined, iconLetter.substr(0, 1) is ''.